### PR TITLE
Updating documentation

### PR DESCRIPTION
**Pull request by ClarytyLLC**
Updating dataset and dataloaders documentation.

---
This change is [Reviewable](https://reviewable.io/reviews/intel-isl/open3d-ml/154)

### CODE REVIEW

File: ml3d/datasets/customdataset.py [hunk 1]
```diff
@@ -49,22 +57,30 @@ def get_data(self, idx):
             feat = np.array(data[:, 3:],
                             dtype=np.float32) if data.shape[1] > 3 else None
             labels = np.zeros((points.shape[0],), dtype=np.int32)
-
-        data = {'point': points, 'feat': feat, 'label': labels}
+            data = {'point': points, 'feat': feat, 'label': labels}
```
Comment (sanskar107) [inline ml3d/datasets/customdataset.py:60]: Can you decrease a level of indentation here?

**Review by sanskar107 — CHANGES_REQUESTED** [review]

File: ml3d/datasets/customdataset.py (hunk 1 above)
Comment (ClarytyLLC) [reply ml3d/datasets/customdataset.py]: I have decreased the indent.

File: ml3d/vis/visualizer.py [hunk 2]
```diff
@@ -13,15 +13,19 @@
...
+                name: The name of the pointcloud array.
+                boxes: The array of pointcloud that define the bounding box.
+        """
 
-        def __init__(self, name, boxes):
-            self.name = name
-            self.boxes = boxes
+    def __init__(self, name, boxes):
+        self.name = name
+        self.boxes = boxes
```
Comment (sanskar107) [inline ml3d/vis/visualizer.py:28]: The indentation level is changed. Can you update this?

**Review by sanskar107 — CHANGES_REQUESTED** [review]

File: ml3d/datasets/customdataset.py (hunk 1 above)
Comment (sanskar107) [reply ml3d/datasets/customdataset.py]: This is still incorrect for line `60`

Comment (germanros1987) [reply ml3d/datasets/customdataset.py]: @ClarytyLLC please, check the indentation of "data". It should be outside the else clause.

**Pull request merged**